### DISCUSSION TITLE
Add password when sharing notes on web

### DISCUSSION THREAD

**Discussion by faithsws** [TriliumNext/Trilium#2916] [Ideas]
Add password when sharing notes on web.
I do not want that anybody can read my shared note on web, could add a password when I share a note on web?
Thus one knows the password only can read the content of note.

**Comment by nicollasricas** [comment]
See if [this](https://github.com/zadam/trilium/wiki/Protected-notes) helps.

You should set the password on _Options -> Password_ before protecting the notes.

**Reply by sigaloid** [reply]
Unfortunately protected notes [cannot be shared](https://github.com/zadam/trilium/wiki/Protected-notes#sharing) currently.

**Reply by faithsws** [reply]
I just want to share notes to somebody who I send any unique password( not my private protect password) to.

**Reply by zadam** [reply]
Protected notes will probably never be shareable, the concept is quite different and IMHO incompatible with sharing.

To password-protect shared notes there would be a different mechanism ...

**Comment by sigaloid** [comment]
Here's a relevant feature request: https://github.com/zadam/trilium/issues/2781
